requirements.txt

```diff
@@ -15,7 +15,8 @@ django-tables2==2.5.3
 django-taggit==3.1.0
 django-timezone-field==5.0
 djangorestframework==3.14.0
-drf-spectacular==0.25.1
+drf-spectacular==0.26.1
+drf-spectacular-sidecar==2023.4.1
```

Comment:
We also need to add `drf-spectacular-sidecar` to `base_requirements.txt` with a URL and a brief description, to ensure it gets updated for future releases.

Reply:
@jeremystretch done.